lib/shoulda/matchers/active_model/numericality_matchers/even_number_matcher.rb

```diff
@@ -20,6 +20,11 @@ def allowed_type
           def diff_to_compare
             2
           end
+
+          def strict
```

Comment:
Comparison matcher is the only matcher that out of the numericality matchers that inherits from `ValidationMatcher` Should I make the others inherit from that matcher as well?